Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -4978,7 +4978,7 @@ void WebPageProxy::commitProvisionalPage(IPC::Connection& connection, FrameIdent
     const auto oldWebPageID = m_webPageID;
     swapToProvisionalPage(provisionalPage.releaseNonNull());
 
-    didCommitLoadForFrame(connection, frameID, WTFMove(frameInfo), WTFMove(request), navigationID, mimeType, frameHasCustomContentProvider, frameLoadType, certificateInfo, usedLegacyTLS, privateRelayed, containsPluginDocument, hasInsecureContent, mouseEventPolicy, userData);
+    didCommitLoadForFrame(connection, frameID, WTFMove(frameInfo), WTFMove(request), navigationID, mimeType, frameHasCustomContentProvider, frameLoadType, certificateInfo, usedLegacyTLS, privateRelayed, String { }, WebCore::ResourceResponseBaseSource::Unknown, containsPluginDocument, hasInsecureContent, mouseEventPolicy, userData);
```

Comment:
Ditto about use of unknown.  Same with the proxy name.  That'll be clearer when they're all in a struct.

Reply:
done.